pkg/sloop/ingress/kubewatcher.go

```diff
@@ -73,6 +89,9 @@ func NewKubeWatcherSource(kubeClient kubernetes.Interface, outChan chan typed.Ku
 		if err != nil {
 			return nil, err
 		}
+
+		kw.refreshCrd = time.NewTicker(crdRefreshInterval)
+		go kw.refershCrdInformers(masterURL, kubeContext)
```

Comment:
here as well 'refreshCRDInformers' ?

Reply:
though technically correct (should capitalize the acronym), the acronym feels like (used as) a word and the camelCase seems appropriate.
Also the number of places that would need to change would make this PR harder to read.

Reply:
the spelling of refresh...